source/extensions/filters/network/redis_proxy/conn_pool_impl.h

```diff
@@ -30,7 +31,7 @@ namespace ConnPool {
 // TODO(mattklein123): Circuit breaking
 // TODO(rshriram): Fault injection
 
-class ConfigImpl : public Config {
+class ConfigImpl : public Common::Redis::Config {
```

Comment:
Should ConfigImpl also be moved to the common library?

Reply:
sure sounds good